example/input/Input__Flag_Spectral

```diff
@@ -0,0 +1,13 @@
+# Level Refinement  Derefinement
```

Comment:
The `Derefinement` column is currently not used. Should we still keep it for future use?

Reply:
I do not think it hurts to leave it there, but if you would rather not have it there, I can of course remove it.

Reply:
It's OK to keep it. But add a comment somewhere that this derefinement threshold is currently not used.